mix.exs

```diff
@@ -83,6 +83,7 @@ defmodule Appsignal.Mixfile do
   defp deps do
     [
       {:httpoison, "~> 0.11"},
+      {:poison, ">= 2.0.0"},
```

Comment:
Phoenix 1.2 and 1.1 depend on poison `"~> 1.5 or ~> 2.0"` and Phoenix 1.0 depends on `"~> 1.3"`. I've done some testing and everything seems to be working in lower versions of Poison, so can we drop this down a bit more to `>= 1.3.0`?